control_freq: 30.0
xy_tolerance: 0.005
theta_tolerance: 0.02
backward_tolerance: 0.0

deviate_distance: 0.10
deviate_angle: 0.2

Kd: 55.0
Kt: 10.0

# Kd: 25.0
# Kt: 2.5

# Kd: 12.0
# Kt: 8.0

# Kd: 4.0
# Kt: 2.5

# Kd: 0.2
# Kt: 0.7

latch_xy: true
linear_compensate: true
overshoot_compensate: true

max_linear_velocity: 0.5
v_threshold: 0.05
velocity_acc: 0.5
velocity_deceleration: 0.3
velocity_deceleration_emergency: 0.3
v_recovery: 0.05

max_angular_velocity: 1.2
w_threshold: 0.2
angular_velocity_acc: 2.0
angular_velocity_deceleration: 1.5
angular_deceleration_emergency: 1.0
w_recovery: 0.2

tolerance_input: [[1,0.1,0.1], [2,0.05,0.1],  [3,0.05,0.1],[5,0.01,0.1], [6,0.05,0.1],  [7,0.02,0.1], [8,0.015,0.02], [12,0.05,0.02], [13,0.01,0.05], [14,0.01,0.05], [15,0.005,0.05], [23,0.01,0.1], [25,0.02,0.1], [28,0.02,0.05], [29,0.02,0.05] ]

#tolerance_input: [[14,0.01,0.1], [19,0.01,0.1], [20,0.01,0.1], [21,0.01,0.1] ]
